src/ApiService/ApiService/Log.cs

```diff
@@ -0,0 +1,172 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.ApplicationInsights.DataContracts;
+
+using System.Diagnostics;
+
+namespace Microsoft.OneFuzz.Service;
+
+public interface ILog {
```

Comment:
why create a new logging interface when ILogger already exist ?

Reply:
because ILogger does not have methods that I'd like to have.

No metrics logging, no Event logging. You cannot pass metrics with Exception if you have any.

Reply:
can we extend iLogger then ?